pay-api/src/pay_api/models/eft_short_name_links.py

```diff
@@ -98,5 +99,6 @@ def from_row(cls, row: EFTShortnameLinks):
                    amount_owing=getattr(row, 'total_owing', None),
                    updated_by=row.updated_by,
                    updated_by_name=row.updated_by_name,
-                   updated_on=row.updated_on
+                   updated_on=row.updated_on,
+                   has_pending_payment=bool(getattr(row, 'invoice_count', 0))
```

Comment:
Used for UI to drive what buttons are available.

E.G. 
- "Cancel Payment" button when there are pending eft_credit_invoice_links
- "Unlink Account" button when there are no pending eft_credit_invoice_links